src/index.test.js

```diff
@@ -2,134 +2,135 @@ import { validate } from "./validate.js";
 
 test("no-alt-text: should return errors", async () => {
   let result = await validate("![]()");
-  expect(result[0]).toBe("Images should have alternate text (alt text)");
+  console.log(result)
+  expect(result).toBe("- Images should have alternate text (alt text) at line 1");
```

Comment:
Nit: This is pre-existing so maybe we want to follow up separately...
I think alt text stands for alternative text, and not alternate text!

Reply:
This is actually text that comes from the linter!